Description: Fix various minor manpage issues and correct section numbering.
 Tools intended for unprivileged users should have their manpages installed
 in section 1 (EG: getenforce/selinuxenabled).
 .
 Also, fix a broken link from security_setenforce.3 to security_getenforce.3
 This change was extracted from the old monolithic Debian patch.
 .
 Tweaked for 2.1.9 by Russell Coker
Author: Kyle Moffett <Kyle.D.Moffett@boeing.com>

---
Forwarded: no
Last-Update: 2011-10-13

--- a/man/man3/security_setenforce.3
+++ b/man/man3/security_setenforce.3
@@ -1 +1 @@
-.so security_getenforce.3
+.so man3/security_getenforce.3.gz
--- a/man/man8/getenforce.8
+++ b/man/man8/getenforce.8
@@ -12,4 +12,4 @@ reports whether SELinux is enforcing, pe
 Dan Walsh, <dwalsh@redhat.com>
 
 .SH "SEE ALSO"
-selinux(8), setenforce(8), selinuxenabled(8)
+selinux(8), setenforce(8), selinuxenabled(1)
--- a/man/man8/selinux.8
+++ b/man/man8/selinux.8
@@ -75,7 +75,7 @@ The best way to relabel the file system
 This manual page was written by Dan Walsh <dwalsh@redhat.com>.
 
 .SH "SEE ALSO"
-booleans(8), setsebool(8), selinuxenabled(8), togglesebool(8), restorecon(8), setfiles(8), ftpd_selinux(8), named_selinux(8), rsync_selinux(8), httpd_selinux(8), nfs_selinux(8), samba_selinux(8), kerberos_selinux(8), nis_selinux(8), ypbind_selinux(8)
+booleans(8), setsebool(8), selinuxenabled(1), togglesebool(8), restorecon(8), setfiles(8), ftpd_selinux(8), named_selinux(8), rsync_selinux(8), httpd_selinux(8), nfs_selinux(8), samba_selinux(8), kerberos_selinux(8), nis_selinux(8), ypbind_selinux(8)
 
 
 .SH FILES
--- a/man/man8/selinuxenabled.8
+++ b/man/man8/selinuxenabled.8
@@ -13,4 +13,4 @@ if SELinux is enabled and 1 if it is not
 Dan Walsh, <dwalsh@redhat.com>
 
 .SH "SEE ALSO"
-selinux(8), setenforce(8), getenforce(8)
+selinux(8), setenforce(8), getenforce(1)
--- a/man/man8/setenforce.8
+++ b/man/man8/setenforce.8
@@ -1,4 +1,4 @@
-.TH "setenforce" "1" "7 April 2004" "dwalsh@redhat.com" "SELinux Command Line documentation"
+.TH "setenforce" "8" "7 April 2004" "dwalsh@redhat.com" "SELinux Command Line documentation"
 .SH "NAME"
 setenforce \- modify the mode SELinux is running in.
 .SH "SYNOPSIS"
@@ -16,4 +16,4 @@ If SELinux is disabled and you want to e
 Dan Walsh, <dwalsh@redhat.com>
 
 .SH "SEE ALSO"
-selinux(8), getenforce(8), selinuxenabled(8)
+selinux(8), getenforce(1), selinuxenabled(1)
--- a/man/man8/togglesebool.8
+++ b/man/man8/togglesebool.8
@@ -1,4 +1,4 @@
-.TH "togglesebool" "1" "26 Oct 2004" "sgrubb@redhat.com" "SELinux Command Line documentation"
+.TH "togglesebool" "8" "26 Oct 2004" "sgrubb@redhat.com" "SELinux Command Line documentation"
 .SH "NAME"
 togglesebool \- flip the current value of a SELinux boolean
 .SH "SYNOPSIS"
--- a/man/man8/selinuxexeccon.8
+++ b/man/man8/selinuxexeccon.8
@@ -1,4 +1,4 @@
-.TH "selinuxexeccon" "1" "14 May 2011" "dwalsh@redhat.com" "SELinux Command Line documentation"
+.TH "selinuxexeccon" "8" "14 May 2011" "dwalsh@redhat.com" "SELinux Command Line documentation"
 .SH "NAME"
 selinuxexeccon \- report SELinux context used for this executable
 
